versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    kubernetes = {
```

Comment:
Since Kubernetes resources are used, we need to specify the required provider and versions https://www.terraform.io/language/providers/requirements#requiring-providers